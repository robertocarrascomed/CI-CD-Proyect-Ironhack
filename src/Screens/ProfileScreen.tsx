// ProfileScreen.tsx
import React from 'react';
import {View, Text} from 'react-native';

const ProfileScreen = () => {
  return (
    <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
      <Text>This is the Profile Screen</Text>
    </View>
  );
};

export default ProfileScreen;
