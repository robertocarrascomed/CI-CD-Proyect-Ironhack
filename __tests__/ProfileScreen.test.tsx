// __tests__/ProfileScreen.test.tsx
import React from 'react';
import renderer from 'react-test-renderer';
import ProfileScreen from '../src/screens/ProfileScreen';

test('ProfileScreen component renders correctly', () => {
  const tree = renderer.create(<ProfileScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
