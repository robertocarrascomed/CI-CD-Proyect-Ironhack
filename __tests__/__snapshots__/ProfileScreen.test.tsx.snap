// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen component renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is the Profile Screen
  </Text>
</View>
`;
